### ISSUE TITLE
Webapp:Runtime parameter does not work properly from powershell

### ISSUE THREAD

**Issue by koolrays** [Azure/azure-cli#14871]
## Describe the bug

**Command Name**
`az webapp create`

**Errors:**
```
'configs'
Traceback (most recent call last):
Local\Temp\pip-unpacked-wheel-kl_cpqvn\knack\cli.py, ln 215, in invoke
azure\cli\core\commands\__init__.py, ln 654, in execute
azure\cli\core\commands\__init__.py, ln 718, in _run_jobs_serially
azure\cli\core\commands\__init__.py, ln 709, in _run_job
azure\cli\command_modules\appservice\commands.py, ln 55, in _polish_bad_errors
azure\cli\core\commands\__init__.py, ln 688, in _run_job
azure\cli\core\commands\__init__.py, ln 325, in __call__
pip-unpacked-wheel-d55550gx\azure\cli\core\__init__.py, ln 779, in default_command_handler
azure\cli\command_modules\appservice\custom.py, ln 167, in create_webapp
azure\cli\command_modules\appservice\custom.py, ln 2527, in update_site_config
KeyError: 'configs'
```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az webapp create --name {} --resource-group {} --plan {} --runtime {}`

## Environment Summary
```
Windows-10-10.0.18362-SP0
Python 3.6.8
Installer: MSI

azure-cli 2.10.1

```

**Comment by ThejaChoudary** [comment]
Assigning it to myself to do repro

**Comment by calvinsID** [comment]
@koolrays Please try using the newest CLI version, the fix should be in 2.11.0
Duplicate with #14758
